src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -138,20 +138,23 @@
         return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder().data(caseData).build());
     }
 
-    @PostMapping(value = "/case-orchestration/notify/consent-order-not-approved", consumes = APPLICATION_JSON_VALUE)
-    @ApiOperation(value = "send e-mail for Consent order not approved.")
+    @PostMapping(value = "/case-orchestration/notify/order-not-approved", consumes = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "send e-mail for consent/contest order not approved.")
     @ApiResponses(value = {
-        @ApiResponse(code = 204, message = "Consent order not approved e-mail sent successfully",
+        @ApiResponse(code = 204, message = "Consent/Contest order not approved e-mail sent successfully",
             response = AboutToStartOrSubmitCallbackResponse.class)})
     public ResponseEntity<AboutToStartOrSubmitCallbackResponse> sendConsentOrderNotApprovedEmail(
         @RequestBody CallbackRequest callbackRequest) {
-        log.info("Received request to send email for 'Consent Order Not Approved' for Case ID: {}", callbackRequest.getCaseDetails().getId());
+        log.info("Received request to send email for 'Consent/Contest Order Not Approved' for Case ID: {}", callbackRequest.getCaseDetails().getId());
         validateCaseData(callbackRequest);
         Map<String, Object> caseData = callbackRequest.getCaseDetails().getData();
 
-        if (isApplicantSolicitorAgreeToReceiveEmails(caseData)) {
+        if (isConsentedApplication(callbackRequest.getCaseDetails()) && isApplicantSolicitorAgreeToReceiveEmails(caseData)) {
```

Comment:
`if (isApplicantSolicitorAgreeToReceiveEmails(caseData)) {
   isConsentedApplication(callbackRequest.getCaseDetails(){
...
} else {
...
}`

This way means you don't have to do the emails check twice making it quicker